client/app.zig

```diff
@@ -359,17 +370,54 @@ fn handleMessages(self: *App) !void {
 
                         log.debug("Head output device updated to {s}", .{@tagName(self.game.head_output_device)});
                         log.debug("Main process PID {d}", .{renderer_init_data.mainProcessId});
+
+                        const formats = comptime std.enums.values(renderite.Shared.TextureFormat);
+
+                        const supported_formats = self.graphics.sampler_supported_formats.unionWith(self.graphics.cubemap_supported_formats);
+                        const supported_formats_len = supported_formats.count();
+
+                        var supported_formats_buf: [formats.len]renderite.Shared.TextureFormat = undefined;
+                        var i: usize = 0;
+                        for (formats) |format| {
+                            if (supported_formats.contains(format)) {
+                                supported_formats_buf[i] = format;
+                                i += 1;
+                            }
+                        }
+
+                        try self.messaging.host.primary.send(.{
+                            .RendererInitResult = .{
+                                .actualOutputDevice = .Screen,
+                                .stereoRenderingMode = &.{},
+                                .isGPUTexturePOTByteAligned = false,
+                                .maxTextureSize = 1024,
```

Comment:
It's a safe bet on desktop with the APIs we're using that this is going to be a supported texture size. We should add a method into GPU to be able to query this later in the future for mobile-type hardware, though. See [this SDL_gpu issue](https://github.com/libsdl-org/SDL/issues/11771).

```suggestion
                                .maxTextureSize = 16384,
```